contents/code/bladelayout.js

```diff
@@ -52,20 +52,19 @@ BladeLayout.prototype.addTile = function() {
             return;
         } else {
             // Divide the screen width evenly between full-height tiles
-            var lastRect = this.tiles[0].rectangle;
-            var newRect = Qt.rect(this.screenRectangle.x,
-                                  lastRect.y,
-                                  (this.screenRectangle.width + this.screenRectangle.x) / (this.tiles.length + 1), 
+            var tileWidth = Math.floor(this.screenRectangle.width / (this.tiles.length + 1));
```

Comment:
This "Math.floor" is the major difference, right?

Because other than that, the calculations should be basically the same? Or is it the lastRect.y vs screenRectangle.y?

Reply:
I'm not really sure to be honest. I basically rewrote the parts that seemed fishy to me and the problem is gone.